docs/README.md

```diff
@@ -77,6 +77,9 @@ In the example above, the [Rich Text](https://vtex.io/docs/components/all/vtex.r
 | Prop name | Type | Description | Default value |
 | --- | --- | --- | --- |
 | `trigger` | `Enum` | Whether the Modal content should be triggered by user click ( `click`) or when the page is fully loaded (`load`)  | `'click'` |
+| `openPerSection` | `Boolean` | Use a cookie to open the modal just once, when the trigger is `load`                                    |  `false`  |
```

Comment:
We avoid using booleans as props in order to make the API cleaner, so I think this could be just an option of `trigger`, something like `'on-load-per-section'` I think